core/src/SYCL/Kokkos_SYCL_Space.cpp

```diff
@@ -125,16 +125,19 @@ DeepCopy<Kokkos::Experimental::SYCLDeviceUSMSpace, Kokkos::HostSpace,
 namespace Kokkos {
 namespace Experimental {
 
-SYCLDeviceUSMSpace::SYCLDeviceUSMSpace() : m_device(SYCL().sycl_device()) {}
+SYCLDeviceUSMSpace::SYCLDeviceUSMSpace()
+    : m_queue(*SYCL().impl_internal_space_instance()->m_queue) {}
+SYCLDeviceUSMSpace::SYCLDeviceUSMSpace(sycl::queue queue) : m_queue(queue) {}
```

Comment:
Should be:

`SYCLDeviceUSMSpace::SYCLDeviceUSMSpace(sycl::queue queue) : m_queue(std::move(queue)) {}`

Reply:
We should really just enable the corresponding `clang-tidy` checks.